cloudbuild.py.yaml

```diff
@@ -24,15 +24,25 @@ steps:
         ./run_test.sh -p
 
   - id: package_js
-    name: "node"
+    name: gcr.io/datcom-ci/node-package:2020-10-05
```

Comment:
can just call it "node" instead of "node-package"

Reply:
fixed